extensions/eclipse-che-theia-plugin-ext/src/node/che-api-service.ts

```diff
@@ -85,6 +85,69 @@ export class CheApiServiceImpl implements CheApiService {
         }
     }
 
+    async generateSshKey(service: string, name: string): Promise<cheApi.ssh.SshPair> {
+        try {
+            const client = await this.wsClient();
+            if (client) {
+                return client.generateSshKey(service, name);
+            }
+
+            throw new Error(`Unable to generate SSH Key for ${service}:${name}`);
+        } catch (e) {
+            throw new Error('Unable to create Che API REST Client');
+        }
+    }
+
+    async createSshKey(sshKeyPair: cheApi.ssh.SshPair): Promise<void> {
+        try {
+            const client = await this.wsClient();
+            if (client) {
+                return client.createSshKey(sshKeyPair);
+            }
+
+            throw new Error(`Unable to create SSH Key`);
+        } catch (e) {
+            throw new Error('Unable to create Che API REST Client');
```

Comment:
if try () catch is only for
```
 await this.wsClient()
```

why not throw an error instead of returning undefined ?
also try / catch block should be just around this.wsClient() if kept like this

note: this comment applies to all get***() methods